test-cases/gutenberg/video.md

```diff
@@ -5,17 +5,32 @@
 
 #### **Precondition**
 
-A site with premium or business plan
+Free or Personal plans are limited to videos that are less than five minutes in length. Currently, there is no size limitation other than the available storage on the user's site.
 
 --------------------------------------------------------------------------------
 
 ##### TC001
 
-**Insert video from device (failing)**
+**Insert video from device (paused or failing)**
 
-Steps are same with [image block TC001](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/image.md#tc001) except for the difference in UI:
+- Add a Video block
+- Tap option ‘Choose from device’ and choose a Video ([precondition applies](#precondition))
+- Disable the internet connection, for example:
+  - By turning on Airplane mode
+  - Throttling the connection with a proxy or third-party app
+- You should see the "Working offline" notice
+- Expect the block's placeholder to show the paused progress bar
+<img src="../../test-cases/resources/../resources/video-upload-progress.jpg" width="260">

```

Comment:
I believe this is incorrect. 

This describes https://github.com/WordPress/gutenberg/issues/59247, but that only applies to Media & Text and should not occur (i.e., a known regression). Do you experience https://github.com/WordPress/gutenberg/issues/59247 for Video blocks as well? I do not on iOS.